rds/aurora/mysql/cluster/Acornfile

```diff
@@ -34,11 +39,12 @@ services: rds: {
 
 jobs: apply: {
 	build: {
-		context: ".."
-		buildArgs: MAIN: "aurora-cluster"
+		context: "../../../"
+		dockerfile: "../../../mysql.Dockerfile"
```

Comment:
Context path changes are a result of the general RDS directory structure refactor done here. Bill and I decided that RDS service acorns would follow the structure of `rds/aurora/$engine/$type`.